src/mame/drivers/eispc.cpp

```diff
@@ -21,13 +21,13 @@
  * On board ports: Beeper,
  * Ports: serial, parallel
  * Internal Options: Up to 640K RAM through add-on RAM card
- * Misc: The hardware was not 100% PC compatible so non BIOS based software would not run. 50.000+ units sold
+ * Misc: The hardware was not 100% PC compatible so non BIOS based software would not always run. 50.000+ units sold
  *
- * TODO:
- * - Add keyboard, first HLE as in pc.cpp and then LLE when the keyboard controller is dumped
- * - Add the on-board FDC and boot DOS 1.xx
- * - Complete the Ericsson 1070 MDA ISA board and test all the graphics modes inclusing 640x400 (aka HR)
+ * TODO:s in selection
```

Comment:
Can you please not do this weird stuff with colons?  This is in no way correct use of a colon in English.  Also "TODO" is not an initialism, so "TODO's" is only correct for a possessive, nto a plural.